docs/hopsworksai/azure/aks_acr_integration.md

```diff
@@ -11,31 +11,31 @@ Hopsworks AKS and ACR integration have four requirements:
+    A user assigned managed identity (managed identity) can be created at the subscription level or to a specific resource group in a subscription. The managed identity is attached to the virtual machines that run inside your subscription (or resource group). Hence, the permissions only apply to services that run within your subscription (or resource group). 
 
-AKS and ACR integration requires some additional permissions to be attached to the managed identity used by the Hopsworks cluster. 
+The AKS and ACR integration requires some permissions to be attached to the managed identity used by the Hopsworks cluster. 
 
-To setup the managed identity, go to the resource group where you will add the managed identity - this should be the same resource group as you deploy Hopsworks in. In your selected resource group, click on the Access Control (IAM) menu item, and then click on add button. In the search dialog, enter "user assigned managed identity", and then select a name for the managed identity, review it and create it.
+If you have already created a *user assigned managed identity* for the storage continue to [Add role assignment to the managed identity](#add-role-assignment-to-the-managed-identity) using this identity.
 
-### Add role assignment at subscription level
+To set up the managed identity, go to the resource group where you will add the managed identity - this should be the same resource group you will deploy Hopsworks in. Click on the *Add* button. In the search dialog, enter *"user assigned managed identity"*.  Click on *Create*. Then give a name to the managed identity and make sure that it is in the Region where you will deploy your cluster. Click on *Review + create*, and click on *Create*.
 
-Next go to the created managed identity in order to create role assignments. Go to azure role assignments and add the following roles:
+### Add role assignment to the managed identity
 
-* AcrPull
+Go to the managed identity created above. Click on *Azure role assignments* in the left column. Click on *Add role assignment*. Select the *Scope* *Resource group* or *Subscription* depending on your preference. Select the *Role* *AcrPull* and click on *Save*.
+
+Repeat the same operation with the following roles:
 * AcrPush
 * AcrDelete
 * Azure Kubernetes Service Contributor Role
 
-The roles can have either scope of your subscription or the specific resource group you are planning to use. A scope of the resource group will restrict the permissions of the roles to the chosen resource group.
-
 !!! warning
-You will also need to attach the Storage Blob Data Owner role to the managed identity, see [Creating and configuring a storage](getting_started.md#step-2-creating-and-configuring-a-storage)
+    You will also need to attach the Storage Blob Data Owner role to the managed identity, see [Creating and configuring a storage](getting_started.md#step-2-creating-and-configuring-a-storage)
```

Comment:
We don't use the Storage Blob Data Owner anymore if i'm not mistaken @smkniazi 
Instead, we create a restrictive storage role 
https://docs.hopsworks.ai/latest/hopsworksai/azure/getting_started/#step-2-creating-and-configuring-a-storage